src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,197 @@
+var malawi_binary = ee.Image("projects/ee-aasareansah/assets/malawi_binary"),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi = ee.FeatureCollection("FAO/GAUL/2015/level0").filter(ee.Filter.eq('ADM0_NAME', 'Malawi')),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
+    Malawi_TA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_TAs"),
+    Admin = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_admin2");
```

Comment:
How are these assets created, is it possible to compute them directly?

Reply:
I got the shapefiles and uploaded them but I'll remove them from the general code since it's specific to Malawi